src/globalComponents.tsx

```diff
@@ -1,33 +1,23 @@
 import React from 'react';
 import exposeComponentRenderer from 'shared/lib/exposeComponentRenderer';
 import RightBar from "./shared/components/rightbar/RightBar";
-import {QueryStore} from "./shared/components/query/QueryStore";
-import QueryModal from "./shared/components/query/QueryModal";
 import QueryAndDownloadTabs from "./shared/components/query/QueryAndDownloadTabs";
+import {QueryStore} from "./shared/components/query/QueryStore";
 
-const queryStore = new QueryStore(window.location.href);
+class GlobalStores {
 
-(window as any).addGenesAndSubmitQuery = queryStore.addGenesAndSubmit.bind(queryStore);
+    public static get queryStore() : QueryStore {
+        return (window as any).globalStores.queryStore;
+    }
+
+}
 
 exposeComponentRenderer('renderRightBar', ()=> {
-    return <RightBar store={queryStore}/>;
+    return <RightBar store={GlobalStores.queryStore} />;
 });
 
-exposeComponentRenderer('renderQuerySelectorInModal', ()=><QueryModal store={queryStore} />);
-
 exposeComponentRenderer('renderQuerySelector', (props:{[k:string]:string|boolean|number})=> {
-    return <QueryAndDownloadTabs {...props} store={queryStore} />;
+    (window as any).addGenesAndSubmitQuery = GlobalStores.queryStore.addGenesAndSubmit.bind(GlobalStores.queryStore);
```

Comment:
@adamabeshouse I put this inside the renderer because the global store isn't available at module's rune time.  Do you see any problem here?